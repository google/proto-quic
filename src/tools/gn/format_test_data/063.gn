source_set("test") {
  a = "a"
  b = "b"
  deps = [
    "//a",
    "//a/a",
    "//a/b",
    "//a:a",
    "//a:b",
    "//b",
    ":a",
    ":b",
    "a",
    "a/a",
    "a/b",
    "a:a",
    "a:b",
    "b",
    a,
    b,
  ]

  public_deps = []
  if (condition) {
    public_deps += [
      "//a",
      "//a/a",
      "//a:a",
      ":a",
      "a",
      "a/a",
      "a:a",
      a,
    ]
  }
}
