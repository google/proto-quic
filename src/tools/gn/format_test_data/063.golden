source_set("test") {
  a = "a"
  b = "b"
  deps = [
    ":a",
    ":b",
    "a",
    "a:a",
    "a:b",
    "a/a",
    "a/b",
    "b",
    "//a",
    "//a:a",
    "//a:b",
    "//a/a",
    "//a/b",
    "//b",
    a,
    b,
  ]

  public_deps = []
  if (condition) {
    public_deps += [
      ":a",
      "a",
      "a:a",
      "a/a",
      "//a",
      "//a:a",
      "//a/a",
      a,
    ]
  }
}
